scripts/bulk-upload-documents.py

```diff
@@ -0,0 +1,40 @@
+"""
+This will scan a directory for files with a filename matching -<supplier_id>_Framework_Agreement
+and upload them to the S3 agreements bucket for <stage> with the file path
+<framework_slug>/agreements/<supplier_id>/<supplier_id>-countersigned-framework-agreement.pdf
+
+Usage:
+    scripts/bulk-upload-countersigned-agreements.py <local_directory> <stage> <framework_slug>
+    <document_type> [--dry-run]
```

Comment:
Yes I guess so. We only have agreements that we'd ever want to bulk-upload at the moment I think? In which case just an example usage with `agreements` should be fine.